src/ScikitLearn.jl

```diff
@@ -0,0 +1,490 @@
+module ScikitLearn_
+
+#> export the new models you're going to define (and nothing else):
+export SVMClassifier, SVMRegressor
+export SVMNuClassifier, SVMNuRegressor
+export SVMLClassifier, SVMLRegressor
+
+#> for all Supervised models:
+import MLJBase
+
+#> for all classifiers:
+using CategoricalArrays
+
+#> import package:
+import ..ScikitLearn: @sk_import
+import ..ScikitLearn
+@sk_import svm: SVC
+@sk_import svm: NuSVC
+@sk_import svm: LinearSVC
+
+@sk_import svm: SVR
+@sk_import svm: NuSVR
+@sk_import svm: LinearSVR
+
+mutable struct SVMClassifier{Any} <: MLJBase.Deterministic{Any}
```

Comment:
why `{Any}` ?

Reply:
The only stict requirement of the parameter `R` in `Deterministic{R}` or `Probabilistic{R}` is that the `fitresult` returned by fit is alway of type `R`. (In tests, one should do `@test fitresult isa fitresult_type(model)` . So `Any` always works. 

To get optimal performance in ensembling, you want `R` to be concrete (the "ensemble" is a `Vector{R}`), although it sometimes has to be a "small" union type (see, eg, DecisionTreeRegressor). If you like, you could take a look at ensembling and see if the type can be inferred. This code (refactored from Koala) is quite old. Finding out `R` is a bit of a pain point. To make `R` concrete in the tree case you have to introduce `target_type` as explicit type parameter for the model struct.

Reply:
ok thanks for the explanation, it makes sense